pkgs/development/python-modules/mdpo/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, polib
+, pymd4c
+}:
+
+buildPythonPackage rec {
+  pname = "mdpo";
```

Comment:
To comply with [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes) please have the commit message name be of the format
```
<pkg-name>: <subject-line>
```
for more examples, please look at https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes

in your case, the commit message should be:
```
python3Packages.mdpo: init at 0.2.36
```